install_config/install/advanced_install.adoc

```diff
@@ -60,6 +60,11 @@ xref:../../install_config/install/rpm_vs_containerized.adoc#install-config-insta
 Containerized Hosts] to ensure that you understand the differences between these
 methods, then return to this topic to continue.
 
+If you are interested in optimizing your installation, including recommendations
```

Comment:
Perhaps change this wording to indicate what might constitute "interest" by saying

For larger scale installs and optimizing install time, see the ...

Perhaps drop the "recommendations for networking", not sure what that's referring to?